addons/compat_gm/functions/fnc_onCutRopes.sqf

```diff
@@ -0,0 +1,22 @@
+#include "script_component.hpp"
+/*
+ * Author: nomisum
+ * Function for animating ropes (first introduced for GM Bo 105)
+ * original onCut seems more like a onStowFRIES
```

Comment:
```suggestion
 * Original onCut seems more like a onStowFRIES.
```
Not sure about this comment, maybe more like "Original onCut runs when stowing FRIES".